src/murmur/CMakeLists.txt

```diff
@@ -91,6 +91,7 @@ target_include_directories(mumble_server_object_lib
 )
 
 target_link_libraries(mumble_server_object_lib PUBLIC shared Qt6::Sql)
+target_link_libraries(mumble_server_object_lib PUBLIC shared CLI11::CLI11)
```

Comment:
I think you only need to link `mumble-server` with `CLI11` as we only need the lib for `main.cpp`